include/AudioBufferView.h

```diff
@@ -97,6 +100,171 @@ class BufferViewData<SampleT, DynamicChannelCount>
 	f_cnt_t m_frames = 0;
 };
 
+template<typename SampleT, proc_ch_t channelCount = DynamicChannelCount>
+class InterleavedFrameIteratorData
```

Comment:
It might be easy to write a quick unit test here. On the other hand, we will likely notice very fast if this will not work.

Reply:
This was moved to a separate PR: #8016

And I've been using it all over the place without any issues, so we should be good.